src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/SaveDefendantResponseTest.java

```diff
@@ -60,6 +61,22 @@ public void shouldReturnNewlyCreatedDefendantResponse() throws Exception {
             .contains(Optional.of(response));
     }
 
+    @Test
+    public void shouldReturnNewlyCreatedPartialDefendantResponse() throws Exception {
```

Comment:
What about cases to confirm:
- staff email is not sent
- citizen notifications are not send

I know we would remove this as soon as we implement missing bits in the future so a bit pointless maybe but at the same time writing tests should be our second nature. Will leave it you you @radoslawn-kainos